Vagrantfile

```diff
@@ -78,13 +87,17 @@ cd /vagrant # This is where the host folder/repo is mounted
 gem install bundler foreman
 bundle install
 
-# Install node modules
+# Disable husky inside vagrant:
+echo "HUSKY=0" >> ~/.bash_profile
+
+# Install node modules, uses:
+# --ignore-scripts to prevent scripts automatically executing inside vagrant
+# --frozen-lockfile to prevent lockfile updates coming from vagrant
 sudo corepack enable
-yarn set version classic
```

Comment:
This is superseded by setting the `packageManager` field in the `package.json`, which corepack automatically detects, by switching to this, we prevent the vagrant instance from creating `.yarnrc` and `.yarn/release/*` files.